src/Hooks/useUploadFile/index.tsx

```diff
@@ -0,0 +1,72 @@
+import { useState, useCallback } from "react";
+import { Area } from "./types";
+
+export const useUploadFile = () => {
+  const imageWidth = 355;
+  const imageHeight = 156;
+  const [url, setUrl] = useState<string>("");
+  const [width, setWidth] = useState<number>(0);
+  const [height, setHeight] = useState<number>(0);
+  const [areas, setAreas] = useState<Area[]>([]);
+  const [data, setData] = useState<any[]>([]);
+
+  const setWH = useCallback(
+    (width: number, height: number) => {
+      setWidth(width);
+      setHeight(height);
+    },
+    [setWidth, setHeight]
```

Comment:
I'm curious why the two setter functions are in the dependency array. Any reason to do that?

Reply:
Just because that sometimes my web develop tools console alerted warning of dependency losing, that include setter functions, for made my console log keeping clean, I put setter functions in dependency first.
But in the end I forget to remove it, This is not necessary

Reply:
Noted. Thank you for the quick reply.